### PR TITLE
Trading Page - Order Ticket: make hover state green orange

### PR DESCRIPTION
**Pull request by rubyweb3**
https://github.com/AugurProject/augur/issues/4731

Screen Shot 2020-01-22 at 12 14 19 AM

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 🌈
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/4kgh633**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

File: packages/augur-ui/src/modules/trading/components/form.styles.less [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 .TradingForm {
   > ul {
-  padding: @size-12 @size-12 0;
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/trading/components/form.styles.less]: this should be reversed, it's breaking the look of the order ticket (keep it as padding: 12 12 0;)

**Review by JohnDanz — CHANGES_REQUESTED** [review]
the dropdown is still working on hover, but I'm assuming the other PR will fix this

File: packages/augur-ui/src/modules/trading/components/form.styles.less (hunk 1 above)
Comment (rubyweb3) [reply packages/augur-ui/src/modules/trading/components/form.styles.less]: Thank you for pointing out. Reverted the change.

**Review by JohnDanz — APPROVED** [review]

**Pull request merged**